float64 pos_x
float64 pos_y
bool add
---
string msg_feedback